Backend-tests.md

```diff
@@ -382,7 +357,7 @@ Notice that the test class inherits from `test_utils.GenericTestBase`, which pro
        return abs(number)
    ```
 
-   Forget for the moment that this function is pointless and think about how to test it. We should test the function by providing some values, some positive and some negative, and checking that their absolute values are returned. We should _not_ test it by mocking `abs()` and making sure it was called correctly.
+   Let's forget for the moment that this function is pointless and just focus on thinking about how to test it. We should test the function by providing some values, some positive and some negative, and checking that their absolute values are returned. We should _not_ test it by mocking `abs()` and making sure it was called correctly.
```

Comment:
(Just flagging the note on https://github.com/U8NWXD/oppia-wiki/pull/20#discussion_r702532905 which I left earlier but separately from this review.)

Reply:
done